cmd/convert.go

```diff
@@ -36,7 +36,8 @@ Note:
   resources.
 
 Example:
-  terraform-validator convert ./example/terraform.tfplan --project my-project
+  terraform-validator convert ./example/terraform.tfplan --project my-project \
+    --folder organization/my-org/project/my-project
```

Comment:
Instead of passing the project, I think folder should just include the parent. ex. `organization/my-org` or `organization/my-org/folder/my-folder`.

Also let's call the flag `--ancestry` instead of `--folder`.